pkg/standalone/run.go

```diff
@@ -35,33 +35,38 @@ const sentryDefaultAddress = "localhost:50001"
-	GRPCPort           int    `env:"DAPR_GRPC_PORT" arg:"dapr-grpc-port"`
-	ConfigFile         string `arg:"config"`
-	Protocol           string `arg:"app-protocol"`
-	Arguments          []string
-	APIListenAddresses string `arg:"dapr-listen-addresses"`
-	EnableProfiling    bool   `arg:"enable-profiling"`
-	ProfilePort        int    `arg:"profile-port"`
-	LogLevel           string `arg:"log-level"`
-	MaxConcurrency     int    `arg:"app-max-concurrency"`
-	PlacementHostAddr  string `arg:"placement-host-address"`
-	ComponentsPath     string `arg:"components-path"`
-	ResourcesPath      string `arg:"resources-path"`
-	AppSSL             bool   `arg:"app-ssl"`
-	MetricsPort        int    `env:"DAPR_METRICS_PORT" arg:"metrics-port"`
-	MaxRequestBodySize int    `arg:"dapr-http-max-request-size"`
-	HTTPReadBufferSize int    `arg:"dapr-http-read-buffer-size"`
-	UnixDomainSocket   string `arg:"unix-domain-socket"`
-	InternalGRPCPort   int    `arg:"dapr-internal-grpc-port"`
-	EnableAppHealth    bool   `arg:"enable-app-health-check"`
-	AppHealthPath      string `arg:"app-health-check-path"`
-	AppHealthInterval  int    `arg:"app-health-probe-interval" ifneq:"0"`
-	AppHealthTimeout   int    `arg:"app-health-probe-timeout" ifneq:"0"`
-	AppHealthThreshold int    `arg:"app-health-threshold" ifneq:"0"`
-	EnableAPILogging   bool   `arg:"enable-api-logging"`
+	AppID            string   `env:"APP_ID" arg:"app-id" yaml:"app_id"`
+	AppPort          int      `env:"APP_PORT" arg:"app-port" yaml:"app_port"`
+	HTTPPort         int      `env:"DAPR_HTTP_PORT" arg:"dapr-http-port" yaml:"dapr_http_port"`
+	GRPCPort         int      `env:"DAPR_GRPC_PORT" arg:"dapr-grpc-port" yaml:"dapr_grpc_port"`
+	ProfilePort      int      `arg:"profile-port" yaml:"profile_port"`
+	Command          []string `yaml:"command"`
+	MetricsPort      int      `env:"DAPR_METRICS_PORT" arg:"metrics-port" yaml:"metrics_port"`
+	UnixDomainSocket string   `arg:"unix-domain-socket" yaml:"unix_domain_socket"`
+	InternalGRPCPort int      `arg:"dapr-internal-grpc-port" yaml:"dapr_internal_grpc_port"`
+	SharedRunConfig  `yaml:",inline"`
+}
+
+// SharedRunConfig represents the application configuration parameters, which can be shared across many apps.
+type SharedRunConfig struct {
+	ConfigFile         string `arg:"config" yaml:"config_file"`
+	AppProtocol        string `arg:"app-protocol" yaml:"app_protocol"`
+	APIListenAddresses string `arg:"dapr-listen-addresses" yaml:"api_listen_addresses"`
+	EnableProfiling    bool   `arg:"enable-profiling" yaml:"enable_profiling"`
+	LogLevel           string `arg:"log-level" yaml:"log_level"`
+	MaxConcurrency     int    `arg:"app-max-concurrency" yaml:"_appmax_concurrency"`
+	PlacementHostAddr  string `arg:"placement-host-address" yaml:"placement_host_address"`
+	ComponentsPath     string `arg:"components-path" yaml:"components_dir"`
+	ResourcesPath      string `arg:"resources-path" yaml:"resources_path"`
```

Comment:
according to the discussion in the issue, `resources_dir_path`?

Reply:
I think we decided `app-dir-path` instead of `app-dir` and `resources-path` instead for `resources-dir` to make it consistent with the actual flag `--resources-path`.